x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -246,6 +255,7 @@ export const calculateEndpointAuthz = (
     canReadBlocklist,
     canWriteEventFilters,
     canReadEventFilters,
+    canExecuteKubernetesCommands,
```

Comment:
Just want to make sure - is this grouping of command available to all license levels?

Although the Console is behind Enterprise license (UI only authz control), the `canExecuteKubernetesCommand` is allow regardless of license the way you have it coded - so a client could use the API to actual create these types of response actions.